saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SpatialRoleLabeling/README.md

```diff
@@ -0,0 +1,111 @@
+# Spatial Role Labeling
+In paper [1], the task of spatial role labeling is introduced and an annotation scheme proposed that is language-independent and facilitates the application of machine learning techniques. The framework consists of a set of spatial roles. For a basic example, in the following sentence:
+
+
+Give me [the gray book]{trajector} [on]{spatialindicator} [the big table]{landmark}.
+
+
+The phrase headed by the token book is referring to a trajector object, the phrase headed by the token table is referring to the role of a landmark and these are related by the spatial expression on denoted as spatial indicator. The spatial indicator (often a preposition) establishes the type of spatial relation.
```

Comment:
phrase headed by the token "book" (in quotation, to show that it is part of the text)
Same for 
 - headed by the token "table"